cwl_utils/graph_split.py

```diff
@@ -47,15 +76,16 @@ def my_represent_none(
             for import_name in imports:
                 rewrite_types(entry, "#{}".format(import_name), False)
         if entry_id == "main":
-            entry_id = "unpacked_{}".format(os.path.basename(sourceIO.name))
-        with open(entry_id, "w", encoding="utf-8") as result_handle:
-            yaml.main.round_trip_dump(
-                entry,
-                result_handle,
-                default_flow_style=False,
-                indent=4,
-                block_seq_indent=2,
-            )
```

Comment:
This needs to show up in `yaml_dump` for those who do not want to use `cwlformat`

Reply:
OK, I'll add an option `--pretty` to use `cwlformat`

Reply:
However, if you don't use `cwlformat`, the output will just be a messy json instead of the expected yaml format.